onebusaway-android/src/main/java/org/onebusaway/android/ui/ArrivalsListFragment.java

```diff
@@ -268,6 +269,23 @@ public View onCreateView(LayoutInflater inflater,
     }
 
     @Override
+    public void onViewStateRestored(@Nullable Bundle savedInstanceState) {
```

Comment:
Could you please add a comment here saying why this is added, and reference the issue number?  Something like "This is required to avoid an NPE when the user rotates their phone while setting a route favorite - see #480".